ras_backstage/resources/reporting_units/get_reporting_unit.py

```diff
@@ -36,6 +38,9 @@ def get(ru_ref):
                                               if ce['id'] in case_collection_exercise_ids
                                               and parse_date(ce['scheduledStartDateTime']) < now]
 
+            respondents = respondents_for_survey.get(survey.get('id'))
+            survey['respondents'] = respondents if respondents else []
```

Comment:
Instead of the if/else here you can change the above line to `respondents_for_survey.get(survey.get('id'), [])`